Add note to vsc include about CLI use (#13385)
aspnet/AspNetCore.Docs: aspnetcore/includes/net-core-prereqs-vsc-2.2.md

* [Visual Studio Code](https://code.visualstudio.com/download)
* [.NET Core SDK 2.2 or later](https://www.microsoft.com/net/download/all)
* [C# for Visual Studio Code (latest version)](https://marketplace.visualstudio.com/items?itemName=ms-vscode.csharp)

> The Visual Studio Code instructions use the .NET Core CLI for ASP.NET Core development functions such as project creation. That means you can follow these instructions on any platform (macOS, Linux, or Windows) and with any code editor. Minor changes may be required if you use something other than Visual Studio Code.

Reviewer: ```suggestion
The Visual Studio Code instructions use the .NET Core CLI for ASP.NET Core development functions such as project creation. You can follow these instructions on any platform (macOS, Linux, or Windows) and with any code editor. Minor workflow changes may be required if using an editor other than Visual Studio Code.
```

Reviewer: I'd suggest the same changes below. Should this text be an include too?

Author: It's only in two places so far, and I hesitate to put includes in includes.
